docs/resources/vdb.md

```diff
@@ -67,6 +67,10 @@ resource "delphix_vdb" "vdb_name" {
 
 * `database_name` - (Optional) The name of the database on the target environment. Defaults to vdb_name.
 
+* `cdb_id` - (Optional) The ID of the container database (CDB) to provision an Oracle Multitenant database into. This corresponds to a Source API object. When this is not set, a new vCDB will be provisioned.
```

Comment:
Not sure if "This corresponds to a Source API object" makes much sense for a terraform user (this refers to the DCT Source API)